app/components/contact-details.hbs

```diff
@@ -1,121 +1,165 @@
-<AuCard as |c|>
-  <c.header>
-    <AuHeading @level="3" @skin="4">
-      {{@title}}
-    </AuHeading>
-  </c.header>
-  <c.content>
-    <AuDataTable
-      id="contactDetails"
-      @content={{this.positions}}
-      @noDataMessage="Geen contactpunten"
-      as |t|
+{{#if this.singlePosition}}
+  <form
+    id="edit-contact-form-single"
+    aria-label="edit-contact-form-single"
+    {{on "submit" (perform this.saveContact)}}
+  >
+    <Position::ContactEditCard
+      @address={{this.editingContact.address}}
+      @primaryContact={{this.editingContact.primaryContact}}
+      @secondaryContact={{this.editingContact.secondaryContact}}
+      @isAddressSearchEnabledInitially={{if
+        (or (not this.editingContact.address.id) this.editingContact.address.addressRegisterUri)
+        true
+        false
+      }}
     >
-      <t.content as |c|>
-        <c.header>
-          <th>Adres</th>
-          <th>Bus</th>
-          <th>Provincie</th>
-          <th>E-mail</th>
-          <th>Primair telefoonummer</th>
-          <th>Secundair telefoonummer</th>
-          <th>Selecteer</th>
-          <th></th>
-        </c.header>
-        <c.body as |contact|>
-          <td>
-            {{contact.primaryContact.contactAddress.street}}
+      <:title>Contactgegevens</:title>
+    </Position::ContactEditCard>
+  </form>
+{{/if}}
```

Comment:
It could be simplified by using a `{{#if} / {{else}} / {{/if}}` no ?

Reply:
indeed! I changed it to if/else where applicable, thanks!